docs/zkapps/how-to-test-a-zkapp.mdx

```diff
@@ -1,6 +1,13 @@
 ---
 title: How to Test a zkApp
 hide_title: true
+description: This guide provides a detailed walkthrough on how to write and run tests for your zkApp using the Jest JavaScript testing framework. It includes creating tests for your smart contract, running tests, creating a local blockchain, deploying a contract locally, and writing integration tests.
```

Comment:
```suggestion
description: Guided steps to write and run tests for your zkApp using the Jest JavaScript testing framework, create and run tests, create a local blockchain, deploy a contract locally, and write integration tests.
```

Reply:
thinking about how we search, let's use the verb "create" instead of the gerund "creating"